src/06_multiple_futures/05_futuresunordered.md

```diff
@@ -0,0 +1,21 @@
+# `FuturesUnordered`
+
+The `FuturesUnordered` struct is a set of futures which may be completed in any order.
+
+```rust,edition2018,ignore
+{{#include ../../examples/06_05_futuresunordered/src/lib.rs:simple}}
+```
+
+Because `FuturesUnordered` implements the `Stream` trait using `.next()` 
+on it will return a `Future` which returns the return value from one of
+the futures inside of an `Option`.
+It will return `None` when all futures are completed.
```

Comment:
I think this could be even more concise, something like

```suggestion
`FuturesUnordered` implements a `Stream` trait over the
output of the futures in any order.
```

The example implies how to use that Stream (and hopefully we have dedicated docs for that too).

Reply:
Yes I agree this is a lot clearer